activerecord/test/cases/arel/factory_methods_test.rb

```diff
@@ -41,6 +41,14 @@ def test_lower
         assert_equal "LOWER", lower.name
         assert_equal [:one], lower.expressions.map(&:expr)
       end
+
+      def test_cast
+        false_node = @factory.create_false
```

Comment:
CI is borken in all database adapters that aren't sqlite because False isn't transformed to `0` in the other databases. Can you change the test to use always `0` instead of the false node?

Reply:
Ok no problem. Converted it into a table/field node object.